pyproject.toml

```diff
@@ -44,8 +44,7 @@ dbt-all = [
     "dbt-athena",
     "dbt-bigquery",
     "dbt-clickhouse",
-    # TODO: https://github.com/astronomer/astronomer-cosmos/issues/1379
-    "dbt-databricks<1.9",
+    "dbt-databricks!=1.9.0",
```

Comment:
would be nice to add a comment here on why we don't allow 1.9.0

Reply:
Added a comment in 5595e0191f6a4cc6c09162ff2f883f61c0684b12